ember_debug/adapters/basic.js

```diff
@@ -1,15 +1,12 @@
 /* globals requireModule */
-var Ember = window.Ember;
-var computed = Ember.computed;
-var $ = Ember.$;
-var RSVP = Ember.RSVP;
-var Promise = RSVP.Promise;
-
-export default Ember.Object.extend({
-  init: function() {
-    var self = this;
-    RSVP.resolve(this.connect(), 'ember-inspector').then(function() {
-      self.onConnectionReady();
+const Ember = window.Ember;
+const { $, A, computed, RSVP, Object } = Ember;
```

Comment:
`Object` has meaning in native javascript. You should rename it to something like `EmberObject`:
`const { $, A, computed, RSVP, Object: EmberObject } = Ember;`